chain/reorg_test.go

```diff
@@ -68,79 +68,29 @@ func TestReorgDiffSubset(t *testing.T) {
 // block is 'a' blocks after the genesis block.  The  main chain has an
 // additional 'b' blocks, the fork has an additional 'c' blocks.  This function
 // returns the forked head, the main head and the common ancestor
```

Comment:
Let's add a precondition that `a > 0` in the comment.  This method should fail more gracefully than before this change when `a==0`(I think it fails [here](https://github.com/filecoin-project/go-filecoin/pull/3063/files#diff-8a85ecf0d5b5586324ba95f4530a6a0dR82) now) but still worth a warning.